spec/factories/json/json_response_factory.rb

```diff
@@ -3,121 +3,118 @@
 
 FactoryBot.define do
   factory :json_build_response_commands, class: ::EtApi::Test::Json::Document do
+    transient do
```

Comment:
This factory has been made more flexible so it is easy to specify the 2nd level objects (i.e. the commands)